store/snapshots/stream.go

```diff
@@ -5,8 +5,7 @@ import (
 	"compress/zlib"
 	"io"
 
-	"cosmossdk.io/errors"
-
+	errorsmod "cosmossdk.io/errors"
```

Comment:
is this linting error? we should try to leave these unchanged

Reply:
Ah ok I was doing those for consistency

Reply:
I thought I was following a pattern, but it was the wrong pattern lol. So I'm changing any import that doesn't have a conflicting import name to be just `cosmossdk.io/errors`. I was confused because I've seen many places in which the old error package was imported as "sdkerrors" even when there wasn't another package with the same name